client/src/components/Feed/PostSocial.js

```diff
@@ -65,24 +66,39 @@ const PostSocial = ({
     );
   };
 
+  const gtmTag = (element) =>
+    GTM.post.prefix + GTM.post[element] + "_" + postId;
```

Comment:
*nit*, template literals  are a bit cleaner imo: 
```suggestion
    `${GTM.post.prefix}${GTM.post[element]}_${postId}`;
```

Reply:
Can certainly change to template literals.